### PR TITLE
ADD to USERS.md Kilo and Sapian

### PR DESCRIPTION
**Pull request by arpagon**
Developer Certificate of Origin
Signed-off-by: Sebastian Rojo <arpagon@gmail.com>

Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

Fixes: #issue-number

ADD to USERS.md Kilo and Sapian

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -385,6 +391,12 @@ Users (Alphabetically)
       L: https://www.concur.com, https://gardener.cloud/, https://github.com/gardener/gardener-extension-networking-cilium
       Q: @dragan (SAP Concur), @docktofuture & @ScheererJ (Gardener)
 
+    * N: Sapian
+      D: Sapian uses Cilium as default CNI in our product DialBox Cloud, DialBox cloud is a Edge Kubernetes cluster using [kilo](https://github.com/squat/kilo) for wirguard mesh connectivity inter-nodes. Cilium is crucial for low latancy on realtime communincations enviroments.
+      U: CNI, Network Policies, Hubble, kube-proxy replacement
+      L: https://sapian.com.co, https://arpagon.co/blog/k8s-edge
```
Comment (pchaigno) [inline USERS.md:403]: These links seem dead :-/

**Review by pchaigno — COMMENTED** [review]
Thanks for the pull request!

There seem to be an issue with the second entry below.

File: USERS.md (hunk 1 above)
Comment (joestringer) [reply USERS.md:403]: When fixing this, please also fix up the spelling: `communincations` -> `communications`.

Comment (arpagon) [reply USERS.md:403]: Thanks. I have already uploaded the link.

Comment (pchaigno) [reply USERS.md:403]: First one is still failing for me. And Joe's comment is still relevant.

**Review by pchaigno — CHANGES_REQUESTED** [review]

**Review by arpagon — COMMENTED** [review]
Thanks, @joestringer and @pchaigno I have already fixed and check booth links. and fix the typo.

File: USERS.md [hunk 2]
```diff
@@ -385,6 +391,12 @@ Users (Alphabetically)
       L: https://www.concur.com, https://gardener.cloud/, https://github.com/gardener/gardener-extension-networking-cilium
       Q: @dragan (SAP Concur), @docktofuture & @ScheererJ (Gardener)
 
+    * N: Sapian
+      D: Sapian uses Cilium as default CNI in our product DialBox Cloud, DialBox cloud is a Edge Kubernetes cluster using [kilo](https://github.com/squat/kilo) for wirguard mesh connectivity inter-nodes. Cilium is crucial for low latancy on realtime communications enviroments.
```
Comment (joestringer) [inline USERS.md]: I missed it the first time, but we also need to fix `wirguard` -> `WireGuard`.

**Review by joestringer — CHANGES_REQUESTED** [review]

File: USERS.md (hunk 2 above)
Comment (pchaigno) [reply USERS.md]: And latancy -> latency.

Comment (arpagon) [reply USERS.md]: Thanks, @joestringer and @pchaigno I have already fixed the spelling in that section.

**Review by joestringer — APPROVED** [review]

**Review by pchaigno — COMMENTED** [review]
Could you squash all commits into one? We should be good to merge after that.

**Comment by joestringer** [comment]
I've squashed the commits & merged. Thanks for the contribution!

**Pull request merged**